jpo-ode-svcs/src/test/java/us/dot/its/jpo/ode/udp/map/MapReceiverTest.java

```diff
@@ -77,7 +77,6 @@ void testMapReceiver() throws IOException {
     // This is necessary because the schema version is set in only one of the OdeMsgMetadata constructors (this should be fixed)
     // and the schema version is set to the static schema version in the constructor. This means that the schema version
     // will be set to 6 for all OdeMsgMetadata objects created in the MapReceiver run method's code path.
```

Comment:
These comments (lines 76-79) seem outdated now that the static schema version is no longer being set. Should we remove or update?

Reply:
Good point. I have removed this since this is no longer relevant.